### DISCUSSION TITLE
Share same files to multiple recipients at once

### DISCUSSION THREAD

**Discussion by ihubgit** [Jacalz/rymdport#105] [Q&A] [answered]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/Jacalz/rymdport/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

Hi, this is a question before anything else : is it possible to stream/share the same files to multiple recipients at once if i share to all the appropriate code or is it first arrived first served, having basically only 1 DL for each file ?

### Describe the solution you'd like to see.

If this isn't possible yet, that would be a neat feature to have in order not to have to go thru the hassle of manually repeting the same setup for each different recipient.

**Comment by Jacalz** [comment, marked as answer]
This not quite possible with how the wormhole protocol works. It is a peer to peer solution and we can't send to multiple peers at once. What you could do is to spin up multiple sends with different codes (we can't create two with the same code) but that wouldn't be very helpful I don't think. You are probably just better off using some other file transfer service in that case. I don't plan on adding support for that.

FYI: Please use [Discussions](https://github.com/Jacalz/rymdport/discussions) for questions. I'll transfer thus this over.
